### ISSUE TITLE
Questions about the weight distribution on loss

### ISSUE THREAD

**Issue by wysqh** [poojahira/image-captioning-bottom-up-top-down#19]
Hi,
I saw the loss backpropagation in train.py is defined as loss = loss1 + 10 * loss2, 

where loss1 is the cross-entropy in the second layer of lstm and loss2 is the multi-label margin loss in the first layer of lstm.

So, my question is: Is there any consideration in view of the weight distribution on loss? why we multiply the loss2 by 10? Or does it actually contain some specific meanings (i.e. max sequence length) or something?

**Comment by poojahira** [comment]
Hi there is no special meaning. I just read a couple of papers and did the weight distribution based on that.
